portfolio/src/main/webapp/index.html

```diff
@@ -21,6 +21,10 @@ <h1>Sara's Portfolio</h1>
       <div>
         <p>Click <a href="/data">here</a> to view content generated by a servlet.</p>
       </div>
+      <div>
+        <button onclick="getGreetingUsingArrow()">Get greeting!</button>
```

Comment:
I would prefer just calling this getGreeting. The fact that it uses the Arrow function is something that is internal to the function and there is no value in exposing it.

Reply:
Done